README.md

```diff
@@ -136,11 +136,22 @@ See [examples on how to use opengpt with different models.](./examples) 🔥
 To do so, you can use the `serve` command:
 
 ```bash
-opengpt serve stabilityai/stablelm-tuned-alpha-3b --precision fp16 --device_map balanced
+opengpt serve yahma/llama-7b-hf --precision fp16 --device_map balanced
```

Comment:
what's the reason to use llama as the default example in our README?

Reply:
Just to show how to use adapter in llama

Reply:
please use a separate section to show the lora usage.

Reply:
DON'T mix all concepts together in a single case. This is very confusing.